tensorflow-framework/src/main/java/org/tensorflow/framework/data/Dataset.java

```diff
@@ -107,6 +123,72 @@ public final Dataset take(long count) {
         tf, this.getVariant(), tf.constant(count), this.getOutputTypes(), this.getOutputShapes());
   }
 
+  /**
+   * Returns a new Dataset which maps a function across all elements from this iterator, on a single
+   * component of each element.
+   *
+   * <p>For example, suppose each element is a `List<Operand<?>>` with 2 components: (features,
+   * labels).
+   *
+   * <p>Calling `iterator.mapOneComponent(0, features -> tf.math.mul(features, tf.constant(2)))`

```

Comment:
update to `dataset.mapOneComponent` (idem for other methods below).

By the way in Java, you want to use `{@code ...}` for code snippets in your Javadoc instead of backquotes.

Reply:
Updated these!